### DISCUSSION TITLE
Show/hide portals on double click

### DISCUSSION THREAD

**Discussion by elediojunior** [Ross-Patterson/Portals-Desktop-Organization#127] [Q&A] [answered]
Hello

Removed the app's ability to show/hide portals on double click?

I noticed that in the new version the option disappeared in the new version, giving way to a shortcut (CTRL+Shift+A), will this be permanent?

Grateful!

**Comment by Ross-Patterson** [comment, marked as answer]
Hi, I added it back in in 3.0.1, if you switch to the beta channel in the settings you will get the new update.
